src/Pretzel.Logic/Commands/BaseParameters.cs

```diff
@@ -18,23 +19,26 @@ public sealed class BaseParameters
 
         public bool Safe { get; private set; }
 
-        [Export("SourcePath")]
         public string Path { get; private set; }
 
-        [Export]
         public IFileSystem FileSystem { get; private set; }
 
         public List<string> CommandArgs { get; private set; }
 
         private BaseParameters(string[] arguments, IFileSystem fileSystem)
+        {
+            SetDefaults(arguments, fileSystem);
```

Comment:
There is no need for a SetDefaults method anymore, no?

Reply:
Yupp, refactored.